
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="CZBioHub HPC User Guide">
      
      
        <meta name="author" content="@griznog,@rc">
      
      
        <link rel="canonical" href="https://hpc.czbiohub.org/hpc-user-guide/3._Software_Stack_Management/d._Software_List/">
      
      
        <link rel="prev" href="../c._Lmod_Modules/">
      
      
        <link rel="next" href="../e._Python_Anaconda_and_Mamba/">
      
      
      <link rel="icon" href="../../images/favicon.svg">
      <meta name="generator" content="mkdocs-1.6.0, mkdocs-material-9.5.27">
    
    
      
        <title>Software List - CZBioHub HPC User Quide</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.6543a935.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../css/extra.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="custom" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#software-list" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="CZBioHub HPC User Quide" class="md-header__button md-logo" aria-label="CZBioHub HPC User Quide" data-md-component="logo">
      
  <img src="../../assets/logo.svg" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CZBioHub HPC User Quide
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Software List
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="CZBioHub HPC User Quide" class="md-nav__button md-logo" aria-label="CZBioHub HPC User Quide" data-md-component="logo">
      
  <img src="../../assets/logo.svg" alt="logo">

    </a>
    CZBioHub HPC User Quide
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Welcome
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" >
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    1. Getting Started
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            1. Getting Started
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../1._Getting_Started/a._Overview/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Introduction to the HPC Cluster Environment
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../1._Getting_Started/b._Connecting_with_SSH/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Connecting with SSH
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../1._Getting_Started/c._Connecting_with_OnDemand/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Connecting with Open OnDemand
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../1._Getting_Started/d._Data_Management/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data and Storage Management
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    2. Working with Slurm
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            2. Working with Slurm
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../2._Working_with_Slurm/a._Introduction_to_Slurm_and_Resource_Scheduling/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    What is a Job Scheduler?
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../2._Working_with_Slurm/b._How_Job_Submission_Works/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    How Job Submission Works
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../2._Working_with_Slurm/c._Using_Interactive_Jobs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Interactive Jobs
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../2._Working_with_Slurm/d._Using_Simple_Batch_Jobs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Batch Jobs
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../2._Working_with_Slurm/e._Using_Batch_Array_Jobs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Submitting Arrays
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../2._Working_with_Slurm/f._Using_Job_Dependencies/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Job Dependencies
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../2._Working_with_Slurm/g._Advanced_Job_Scripting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Advanced Job Scripts
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../2._Working_with_Slurm/h._Managing_Slurm_Jobs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Job Management
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../2._Working_with_Slurm/i._Using_the_Slurm_Rest_API/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Slurm REST API
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" checked>
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    3. Software Stack Management
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            3. Software Stack Management
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../a._Software_Stack_Overview/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Overview
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../b._Operating_System_Packages/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Operating System
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../c._Lmod_Modules/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Lmod modules
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Software List
  </span>
  

      </a>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../e._Python_Anaconda_and_Mamba/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    The Conda Module
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../f._Self-Managed_Software_and_Applications/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Self-managed software
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    4. Storage and Data Management
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            4. Storage and Data Management
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../4._Storage_and_Data_Management/a._Data_Transfer/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Data Transfer
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../4._Storage_and_Data_Management/b._Storage_Namespaces/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Introduction
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    5. Site and Cluster Infrastructure
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            5. Site and Cluster Infrastructure
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../5._Site_and_Cluster_Infrastructure/a._Overview_of_HPC_Sites_and_Reseources/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Sites Overview
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../5._Site_and_Cluster_Infrastructure/b._Site_BiohubSF.Computational_Infrastructure/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Bruno Infrastructure
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" >
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    6. Recipes for Common Tasks
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            6. Recipes for Common Tasks
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../6._Recipes_for_Common_Tasks/a._Recipes/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Recipes
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../6._Recipes_for_Common_Tasks/b._using_pigz/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tutorial: Using pigz Command on an HPC System
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_8" >
        
          
          <label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    7.Frequently Asked Questions
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_8_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_8">
            <span class="md-nav__icon md-icon"></span>
            7.Frequently Asked Questions
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../7.Frequently_Asked_Questions/a._FAQ_How_do_I_use_NoMachine/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    How do I use NoMachine?
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../7.Frequently_Asked_Questions/b._FAQ_Adding_a_new_question_to_the_FAQ_section/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Adding a new question to FAQ
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../7.Frequently_Asked_Questions/c._FAQ_This_thing_used_to_work_but_is_broken_now/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Something used to work, but broken now
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../7.Frequently_Asked_Questions/d._FAQ_Why_is_my_HOME_quota_so_small/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Why is home so small
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../7.Frequently_Asked_Questions/e._FAQ_Why_no_sudo_access/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Why cant I use sudo?
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_9" >
        
          
          <label class="md-nav__link" for="__nav_9" id="__nav_9_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    A. Contact Us
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_9_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_9">
            <span class="md-nav__icon md-icon"></span>
            A. Contact Us
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../A._Contact_Us/a._Contact_Us/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Contact Us
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


<h1 id="software-list">Software List</h1>
<table>
<thead>
<tr>
<th>Software</th>
<th>Version(s)</th>
<th>Description</th>
<th>Link</th>
</tr>
</thead>
<tbody>
<tr>
<td>BEAGLE</td>
<td>BEAGLE/2023.06.06</td>
<td>BEAGLE is a high-performance library that performs core calculations for Bayesian and Maximum Likelihood phylogenetics, utilizing GPUs for speed.</td>
<td></td>
</tr>
<tr>
<td>BEAST</td>
<td>BEAST/1.10.4</td>
<td>BEAST is a software package for phylogenetic analysis with an emphasis on time-scaled trees.</td>
<td></td>
</tr>
<tr>
<td>BEAST2</td>
<td>BEAST2/2.7.4</td>
<td>BEAST 2 is a cross-platform program for Bayesian phylogenetic analysis of molecular sequences.</td>
<td></td>
</tr>
<tr>
<td>FragPipe</td>
<td>FragPipe/20.0, FragPipe/21.1, FragPipe/22.0</td>
<td>FragPipe is a Java GUI for a suite of tools enabling comprehensive analysis of mass spectrometry-based proteomics data.</td>
<td></td>
</tr>
<tr>
<td>MrBayes</td>
<td>MrBayes/git-d5aa8ed</td>
<td>MrBayes is a program for Bayesian inference and model choice across various phylogenetic and evolutionary models, using MCMC methods.</td>
<td></td>
</tr>
<tr>
<td>R</td>
<td>R/4.1, R/4.2, R/4.3, R/4.4</td>
<td>R is a free software environment for statistical computing and graphics.</td>
<td></td>
</tr>
<tr>
<td>STAR</td>
<td>STAR/default, STAR/2.7.10b, STAR/2.7.11b</td>
<td>Spliced Transcripts Alignment to a Reference.</td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23104886">Link</a></td>
</tr>
<tr>
<td>StdEnv</td>
<td>StdEnv</td>
<td></td>
<td></td>
</tr>
<tr>
<td>alphafold</td>
<td>alphafold/2.1.0, alphafold/2.2.0</td>
<td>AlphaFold is an AI program developed by DeepMind for predicting protein structures using deep learning.</td>
<td></td>
</tr>
<tr>
<td>anaconda</td>
<td>anaconda/latest, anaconda/2021_09_16, anaconda/2022.05, anaconda/2023.03</td>
<td>Base Anaconda python environment.</td>
<td></td>
</tr>
<tr>
<td>aria2</td>
<td>aria2/1.37.0</td>
<td>aria2 is a lightweight command-line download utility supporting multiple protocols and sources, manipulable via JSON-RPC and XML-RPC interfaces.</td>
<td></td>
</tr>
<tr>
<td>awscli</td>
<td>awscli/v2</td>
<td>AWS Command Line Interface.</td>
<td><a href="https://docs.aws.amazon.com/cli/index.html">Link</a></td>
</tr>
<tr>
<td>beagle</td>
<td>beagle/2023.06.06</td>
<td>BEAGLE is a high-performance library that performs core calculations for Bayesian and Maximum Likelihood phylogenetics, utilizing GPUs for speed.</td>
<td></td>
</tr>
<tr>
<td>beast</td>
<td>beast/1.10.4</td>
<td>BEAST is a software package for phylogenetic analysis with an emphasis on time-scaled trees.</td>
<td></td>
</tr>
<tr>
<td>beast2</td>
<td>beast2/2.7.4</td>
<td>BEAST 2 is a cross-platform program for Bayesian phylogenetic analysis of molecular sequences.</td>
<td></td>
</tr>
<tr>
<td>bowtie2</td>
<td>bowtie2/2.1.0, bowtie2/2.5.3</td>
<td>Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences.</td>
<td></td>
</tr>
<tr>
<td>btop</td>
<td>btop/1.3.0</td>
<td>Btop</td>
<td></td>
</tr>
<tr>
<td>caddy</td>
<td>caddy/2.4.6, caddy/2.6.4</td>
<td>Caddy 2 is a powerful, enterprise-ready, open source web server with automatic HTTPS written in Go.</td>
<td></td>
</tr>
<tr>
<td>cellprofiler</td>
<td>cellprofiler/4.2.1</td>
<td>CellProfiler is free, open-source software designed to enable biologists to quantitatively measure phenotypes from thousands of images automatically.</td>
<td></td>
</tr>
<tr>
<td>cellranger</td>
<td>cellranger/7.0.1</td>
<td>Cell Ranger is a set of analysis pipelines for processing Chromium single cell data to align reads, generate feature-barcode matrices, perform clustering and other secondary analysis.</td>
<td></td>
</tr>
<tr>
<td>cellranger-atac</td>
<td>cellranger-atac/2.1.0</td>
<td>Chromium Single Cell ATAC analyzes thousands of unique open chromatin fragments genome-wide with single cell resolution.</td>
<td></td>
</tr>
<tr>
<td>cistem</td>
<td>cistem/b21db55, cistem/1.0.0</td>
<td>cisTEM</td>
<td></td>
</tr>
<tr>
<td>cmake</td>
<td>cmake/3.28.2</td>
<td>CMake is a powerful, comprehensive solution for managing the software build process.</td>
<td></td>
</tr>
<tr>
<td>comp_micro</td>
<td>comp_micro/default</td>
<td>Module for the computational microscopy group (Shalin Mehta).</td>
<td></td>
</tr>
<tr>
<td>cryosparc</td>
<td>cryosparc/4.4.0-motioncor2-1.6.4, cryosparc/4.4.1-motioncor2-1.6.4, cryosparc/4.5.1-motioncor2-1.6.4</td>
<td>CryoSparc</td>
<td></td>
</tr>
<tr>
<td>ctffind</td>
<td>ctffind/4.1.14</td>
<td>Mindell JA, Grigorieff N. 2003. Accurate determination of local defocus and specimen tilt in electron microscopy. J Struct Biol. 142:334-47.</td>
<td></td>
</tr>
<tr>
<td>cuda</td>
<td>cuda/10.2.89_440.33.01, cuda/11.6.2_510.47.03, cuda/11.7.0_515.43.04, cuda/11.7.1_515.65.01, cuda/11.8.0_520.61.05, cuda/12.0.0_525.60.13, ...</td>
<td>The NVIDIA® CUDA® Toolkit provides a development environment for creating high performance GPU-accelerated applications.</td>
<td></td>
</tr>
<tr>
<td>cudnn</td>
<td>cudnn/8.7.0.84_cuda10, cudnn/8.8.1.3_cuda11, cudnn/8.8.1.3_cuda12, cudnn/8.9.7.29_cuda11, cudnn/8.9.7.29_cuda12, cudnn/9.0.0.3_cuda12</td>
<td>The NVIDIA CUDA® Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks.</td>
<td></td>
</tr>
<tr>
<td>czii</td>
<td>czii/default</td>
<td>Add CZII group managed conda envs and modules.</td>
<td></td>
</tr>
<tr>
<td>data.science</td>
<td>data.science/default</td>
<td>Module for the Data Science group.</td>
<td></td>
</tr>
<tr>
<td>diann</td>
<td>diann/1.8.1</td>
<td>A universal automated software suite for DIA proteomics data analysis.</td>
<td></td>
</tr>
<tr>
<td>duc</td>
<td>duc/1.4.4</td>
<td>Duc is a collection of tools for inspecting and visualizing disk usage.</td>
<td><a href="https://duc.zevv.nl/">Link</a></td>
</tr>
<tr>
<td>elfindo</td>
<td>elfindo/0.9.5</td>
<td>A parallel find tool for Linux.</td>
<td><a href="https://github.com/breuner/elfindo/">Link</a></td>
</tr>
<tr>
<td>emacs</td>
<td>emacs/29.2</td>
<td>GNU Emacs</td>
<td></td>
</tr>
<tr>
<td>ffmpeg</td>
<td>ffmpeg/7.0</td>
<td>FFmpeg</td>
<td></td>
</tr>
<tr>
<td>fm-index</td>
<td>fm-index/1.0</td>
<td>A simple C++ based FM-Index implementation.</td>
<td></td>
</tr>
<tr>
<td>fpart</td>
<td>fpart/1.6.1</td>
<td>Fpart is a Filesystem partitioner. It helps you sort file trees and pack them into partitions.</td>
<td><a href="https://www.fpart.org/">Link</a></td>
</tr>
<tr>
<td>fragpipe</td>
<td>fragpipe/20.0, fragpipe/21.1, fragpipe/22.0</td>
<td>FragPipe is a Java GUI for a suite of tools enabling comprehensive analysis of mass spectrometry-based proteomics data.</td>
<td></td>
</tr>
<tr>
<td>gcc</td>
<td>gcc/11.3</td>
<td>The GNU Compiler Collection.</td>
<td></td>
</tr>
<tr>
<td>gurobi</td>
<td>gurobi/11.0.2</td>
<td>Gurobi Optimizer</td>
<td></td>
</tr>
<tr>
<td>hpcx</td>
<td>hpcx/2.19</td>
<td>NVIDIA HPC-X toolkit</td>
<td></td>
</tr>
<tr>
<td>httm</td>
<td>httm/0.38.0</td>
<td>HTTM prints the size, date, and corresponding locations of available unique versions of files residing on snapshots.</td>
<td><a href="https://github.com/kimono-koans/httm">Link</a></td>
</tr>
<tr>
<td>hwloc</td>
<td>hwloc/2.10.0</td>
<td>The Hardware Locality (hwloc) software project aims at easing the process of discovering hardware resources in parallel architectures.</td>
<td></td>
</tr>
<tr>
<td>iqtree</td>
<td>iqtree/2.2.2.6</td>
<td>Efficient software for phylogenomic inference.</td>
<td></td>
</tr>
<tr>
<td>jacobo_group</td>
<td>jacobo_group/default</td>
<td>Module for the Quantitative Tissue Morphogenesis group (Adrian Jacobo).</td>
<td></td>
</tr>
<tr>
<td>julia</td>
<td>julia/default, julia/1.7.3, julia/1.9.4, julia/1.10.0</td>
<td>The Julia Programming Language.</td>
<td><a href="https://julialang.org/">Link</a></td>
</tr>
<tr>
<td>mafft</td>
<td>mafft/7.520</td>
<td>Multiple alignment program for amino acid or nucleotide sequences.</td>
<td></td>
</tr>
<tr>
<td>mamba</td>
<td>mamba/23.1.0-3-pypy3, mamba/23.1.0-3</td>
<td>Base Mamba python environment.</td>
<td></td>
</tr>
<tr>
<td>maxquant</td>
<td>maxquant/2.0.3.1</td>
<td>MaxQuant is a quantitative proteomics software package designed for analyzing large mass-spectrometric data sets.</td>
<td></td>
</tr>
<tr>
<td>micro-manager</td>
<td>micro-manager/test</td>
<td></td>
<td></td>
</tr>
<tr>
<td>minimap2</td>
<td>minimap2/2.26</td>
<td>Minimap2 is a versatile sequence alignment program that aligns DNA or mRNA sequences against a large reference database.</td>
<td></td>
</tr>
<tr>
<td>mongodb</td>
<td>mongodb/7.0.8</td>
<td>MongoDB is a general-purpose document database.</td>
<td></td>
</tr>
<tr>
<td>mongosh</td>
<td>mongosh/2.2.3</td>
<td>The MongoDB Shell, mongosh, is a JavaScript and Node.js REPL environment for interacting with MongoDB deployments in Atlas, locally, or on another remote host.</td>
<td></td>
</tr>
<tr>
<td>motioncor2</td>
<td>motioncor2/1.6.4</td>
<td>This program corrects whole frame image motion recorded with dose fractionated image stack.</td>
<td></td>
</tr>
<tr>
<td>motioncor3</td>
<td>motioncor3/1.1.1</td>
<td>This program corrects whole frame image motion recorded with dose fractionated image stack.</td>
<td></td>
</tr>
<tr>
<td>mpifileutils</td>
<td>mpifileutils/0.11.1</td>
<td>The mpiFileUtils suite offers MPI-based tools for tasks like copy, remove, and compare for such datasets, delivering performance speedup over their single-process counterparts.</td>
<td><a href="https://mpifileutils.readthedocs.io">Link</a></td>
</tr>
<tr>
<td>mrbayes</td>
<td>mrbayes/git-d5aa8ed</td>
<td>MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. MrBayes uses MCMC methods to estimate the posterior distribution of model parameters.</td>
<td></td>
</tr>
<tr>
<td>mustem-gpu</td>
<td>mustem-gpu/5.3</td>
<td>Open source version of the MuSTEM multislice electron microscopy simulation code, developed at the University of Melbourne.</td>
<td></td>
</tr>
<tr>
<td>nano</td>
<td>nano/7.2</td>
<td>GNU Nano</td>
<td></td>
</tr>
<tr>
<td>neovim</td>
<td>neovim/0.10.0</td>
<td>NeoVIM</td>
<td></td>
</tr>
<tr>
<td>nextflow</td>
<td>nextflow/current</td>
<td>Nextflow is a reactive workflow framework and a programming DSL that eases the writing of data-intensive computational pipelines.</td>
<td></td>
</tr>
<tr>
<td>nextpyp</td>
<td>nextpyp/0.5</td>
<td>nextPYP</td>
<td></td>
</tr>
<tr>
<td>nsight</td>
<td>nsight/2024.1.1.59</td>
<td>A system-wide performance analysis tool from NVIDIA</td>
<td></td>
</tr>
<tr>
<td>nvhpc-sdk</td>
<td>nvhpc-sdk/23.11, nvhpc-sdk/24.3</td>
<td>The NVIDIA HPC SDK includes the compilers, libraries, and software tools essential for maximizing developer productivity and performance of HPC applications.</td>
<td></td>
</tr>
<tr>
<td>omnitrace</td>
<td>omnitrace/1.11.1</td>
<td>Application Profiling, Tracing, and Analysis from AMD</td>
<td></td>
</tr>
<tr>
<td>openmpi</td>
<td>openmpi/4.1.6, openmpi/5.0.3</td>
<td>The Open MPI Project is an open source Message Passing Interface implementation maintained by a consortium of academic, research, and industry partners.</td>
<td></td>
</tr>
<tr>
<td>oracle-java</td>
<td>oracle-java/jdk-17.0.1, oracle-java/jdk-17.0.6, oracle-java/jdk-19.0.1, oracle-java/jdk-20</td>
<td>Java is a programming language and computing platform first released by Sun Microsystems in 1995.</td>
<td></td>
</tr>
<tr>
<td>papi</td>
<td>papi/7.1.0</td>
<td>The Performance Application Programming Interface (PAPI) supplies a consistent interface and methodology for collecting performance counter information.</td>
<td><a href="https://icl.utk.edu/papi/">Link</a></td>
</tr>
<tr>
<td>proot</td>
<td>proot/current</td>
<td>chroot, mount --bind, and binfmt_misc without privilege/setup for Linux.</td>
<td></td>
</tr>
<tr>
<td>pycharm</td>
<td>pycharm/community-2022.2</td>
<td>The Python IDE for Professional Developers</td>
<td><a href="https://www.jetbrains.com/pycharm/">Link</a></td>
</tr>
<tr>
<td>r</td>
<td>r/4.1, r/4.2, r/4.3, r/4.4</td>
<td>R is a free software environment for statistical computing and graphics.</td>
<td></td>
</tr>
<tr>
<td>raxml-ng</td>
<td>raxml-ng/1.2.1</td>
<td>RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion.</td>
<td></td>
</tr>
<tr>
<td>raxml-ng-mpi</td>
<td>raxml-ng-mpi/1.2.1</td>
<td>MPI Version: RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion.</td>
<td></td>
</tr>
<tr>
<td>rclone</td>
<td>rclone/1.65.2</td>
<td>Rclone is a command-line program to manage files on cloud storage.</td>
<td></td>
</tr>
<tr>
<td>relion</td>
<td>relion/master, relion/ver3.1-git-CU80, relion/ver3.1-git-CU86, relion/ver5.0-git-CU80, relion/ver5.0-git-CU86, relion/4.0.1-CU80, relion/4.0.1-CU86, ...</td>
<td>RELION (REgularised LIkelihood OptimisatioN) is a software package employing an empirical Bayesian approach for electron cryo-microscopy (cryo-EM) structure determination.</td>
<td></td>
</tr>
<tr>
<td>relion-turbo</td>
<td>relion-turbo/ver5.0-git-CU80, relion-turbo/ver5.0-git-CU86, relion-turbo/ver5.0-git-CU90</td>
<td>RELION Turbo (REgularised LIkelihood OptimisatioN) is a software package employing an empirical Bayesian approach for electron cryo-microscopy (cryo-EM) structure determination.</td>
<td></td>
</tr>
<tr>
<td>resmap</td>
<td>resmap/1.1.4</td>
<td>ResMap (Resolution Map) is a Python (NumPy/SciPy) application with a Tkinter GUI for computing the local resolution of 3D density maps in structural biology, primarily cryo-EM.</td>
<td></td>
</tr>
<tr>
<td>royerlab</td>
<td>royerlab/default</td>
<td>Module for the Royer Lab.</td>
<td></td>
</tr>
<tr>
<td>rstudio</td>
<td>rstudio/2024.04.1+748</td>
<td>Rstudio is an integrated development environment (IDE) for R.</td>
<td></td>
</tr>
<tr>
<td>rsync</td>
<td>rsync/default, rsync/3.2.7</td>
<td>Rsync is a fast and versatile file copying tool for both remote and local files.</td>
<td></td>
</tr>
<tr>
<td>ruse</td>
<td>ruse/2.0</td>
<td>Ruse: A command line tool to measure process resource usage.</td>
<td></td>
</tr>
<tr>
<td>rust</td>
<td>rust/1.77.2</td>
<td>Rust is a fast and memory-efficient programming language with no runtime or garbage collector, suitable for performance-critical services and embedded devices.</td>
<td><a href="https://www.rust-lang.org/">Link</a></td>
</tr>
<tr>
<td>samtools</td>
<td>samtools/1.18, samtools/1.19</td>
<td>Samtools is a suite of programs for interacting with high-throughput sequencing data.</td>
<td></td>
</tr>
<tr>
<td>schrodinger</td>
<td>schrodinger/2021.04</td>
<td>Schrodinger Suites is a leading commercial application for computational chemistry. The Biohub has licensed the biologics suite.</td>
<td></td>
</tr>
<tr>
<td>slurm</td>
<td>slurm/current</td>
<td>Slurm provides commands and APIs to schedule work on the CZBioHub Cluster Resources.</td>
<td></td>
</tr>
<tr>
<td>spaceranger</td>
<td>spaceranger/2.1.0</td>
<td>Space Ranger is a set of analysis pipelines that process Visium Spatial Gene Expression data with brightfield and fluorescence microscope images.</td>
<td></td>
</tr>
<tr>
<td>spid</td>
<td>spid/master</td>
<td>This package implements a simple alignment and SNP-calling pipeline for detecting viral and bacterial infectious disease outbreak clusters.</td>
<td></td>
</tr>
<tr>
<td>squashfs-tools</td>
<td>squashfs-tools/4.6.1</td>
<td>SQUASHFS - A squashed read-only filesystem for Linux.</td>
<td></td>
</tr>
<tr>
<td>star</td>
<td>star/default, star/2.7.10b, star/2.7.11b</td>
<td>Spliced Transcripts Alignment to a Reference.</td>
<td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23104886">Link</a></td>
</tr>
</tbody>
</table>
<p><em>Note: AI Generated by ChatGPT-4o model, with human edits</em></p>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2024 Chan Zuckerberg Biohub Network.
    </div>
  
  
</div>
      
        <div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://www.czbiohub.org" target="_blank" rel="noopener" title="www.czbiohub.org" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 6.5.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M352 256c0 22.2-1.2 43.6-3.3 64H163.3c-2.2-20.4-3.3-41.8-3.3-64s1.2-43.6 3.3-64h185.4c2.2 20.4 3.3 41.8 3.3 64zm28.8-64h123.1c5.3 20.5 8.1 41.9 8.1 64s-2.8 43.5-8.1 64H380.8c2.1-20.6 3.2-42 3.2-64s-1.1-43.4-3.2-64zm112.6-32H376.7c-10-63.9-29.8-117.4-55.3-151.6 78.3 20.7 142 77.5 171.9 151.6zm-149.1 0H167.7c6.1-36.4 15.5-68.6 27-94.7 10.5-23.6 22.2-40.7 33.5-51.5C239.4 3.2 248.7 0 256 0s16.6 3.2 27.8 13.8c11.3 10.8 23 27.9 33.5 51.5 11.6 26 20.9 58.2 27 94.7zm-209 0H18.6c30-74.1 93.6-130.9 172-151.6-25.5 34.2-45.3 87.7-55.3 151.6zM8.1 192h123.1c-2.1 20.6-3.2 42-3.2 64s1.1 43.4 3.2 64H8.1C2.8 299.5 0 278.1 0 256s2.8-43.5 8.1-64zm186.6 254.6c-11.6-26-20.9-58.2-27-94.6h176.6c-6.1 36.4-15.5 68.6-27 94.6-10.5 23.6-22.2 40.7-33.5 51.5-11.2 10.7-20.5 13.9-27.8 13.9s-16.6-3.2-27.8-13.8c-11.3-10.8-23-27.9-33.5-51.5zM135.3 352c10 63.9 29.8 117.4 55.3 151.6-78.4-20.7-142-77.5-172-151.6h116.7zm358.1 0c-30 74.1-93.6 130.9-171.9 151.6 25.5-34.2 45.2-87.7 55.3-151.6h116.6z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": [], "search": "../../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../../assets/javascripts/bundle.ad660dcc.min.js"></script>
      
    
  </body>
</html>